insights/specs/default.py

```diff
@@ -596,6 +596,7 @@ def sap_sid_name(broker):
     sockstat = simple_file("/proc/net/sockstat")
     softnet_stat = simple_file("proc/net/softnet_stat")
     software_collections_list = simple_command('/usr/bin/scl --list')
+    spamassassin_channels = simple_command("/bin/grep '^\\s*CHANNELURL=' /etc/mail/spamassassin/channel.d/*.conf")
```

Comment:
@skontar there is no shell to interpret the `*.conf` file glob so this command won't work.